CONTRIBUTION.md

```diff
@@ -1,5 +1,33 @@
 # Contribution Guide
 
+## Environment setup
+
+### With docker
+
+1. Fork the repository and `git clone` to your local machine.
+2. Build and access to docker container
+
+```
+docker-compose build ubuntu
+docker-compose run --rm ubuntu bash
+```
+
+In case you want custom ruby or duckdb versions, use `--build-arg` options
+```
+docker-compose build ubuntu --build-arg RUBY_VERSION=3.1.3 --build-arg DUCKDB_VERSION=0.6.0
```

Comment:
@lamphanqg 
Thank you for adding `--build-arg` option.

But I got the following error in my Ubuntu.
Could you fix it?

```
$ docker-compose build ubuntu
Building ubuntu
[+] Building 0.1s (2/2) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                              0.1s
 => => transferring dockerfile: 609B                                                                                                                                              0.0s
 => [internal] load .dockerignore                                                                                                                                                 0.1s
 => => transferring context: 2B                                                                                                                                                   0.0s
Dockerfile:2
--------------------
   1 |     ARG RUBY_VERSION=3.2.1
   2 | >>> FROM ruby:${RUBY_VERSION}
   3 |
   4 |     ARG DUCKDB_VERSION=0.6.1
--------------------
ERROR: failed to solve: failed to parse stage name "ruby:": invalid reference format
ERROR: Service 'ubuntu' failed to build : Build failed
```

My environemnt is
$ cat /etc/os-release | grep PRETTY_NAME
PRETTY_NAME="Ubuntu 20.04.5 LTS"
$ docker-compose --version
docker-compose version 1.29.2, build 5becea4c
$ docker --version
Docker version 23.0.1, build a5ee5b1
```

Reply:
Oh, I tried `docker compose` instead of `docker-compose` and the above error was fixed.

But got the following error.

```
 => [11/12] RUN bundle install                                                                                                                                                    2.2s
 => ERROR [12/12] RUN rake build                                                                                                                                                  2.0s
------
 > [12/12] RUN rake build:
#0 0.564 /usr/local/bin/ruby -I. -r.rake-compiler-siteconf.rb ../../../../ext/duckdb/extconf.rb
#0 0.564 cd tmp/x86_64-linux/duckdb_native/3.2.1
#0 0.631 checking for duckdb.h... yes
#0 0.810 checking for duckdb_pending_prepared() in duckdb.h... no
#0 0.973 checking for duckdb_pending_prepared() in -lduckdb... yes
#0 1.234 checking for duckdb_append_data_chunk() in duckdb.h... yes
#0 1.441 checking for duckdb_value_string() in duckdb.h... yes
#0 1.652 checking for duckdb_free() in duckdb.h... yes
#0 1.860 creating Makefile
#0 1.864 cd -
#0 1.864 cd tmp/x86_64-linux/duckdb_native/3.2.1
#0 1.864 /usr/bin/gmake
#0 1.868 linking shared-object duckdb/duckdb_native.so
#0 2.022 /usr/bin/gmake install target_prefix=
#0 2.022 cd -
#0 2.025 /usr/bin/install -c -m 0755 duckdb_native.so /home/suke/mywork/ruby-duckdb/lib/duckdb
#0 2.026 /usr/bin/install: cannot create regular file '/home/suke/mywork/ruby-duckdb/lib/duckdb': No such file or directory
#0 2.026 gmake: *** [Makefile:193: install-so] Error 1
#0 2.026 rake aborted!
#0 2.026 Command failed with status (2): [/usr/bin/gmake install target_prefix=...]
#0 2.026 /usr/local/bundle/gems/rake-compiler-1.2.0/lib/rake/extensiontask.rb:160:in `block in define_compile_tasks'
#0 2.026 Tasks: TOP => build => compile => compile:x86_64-linux => compile:duckdb_native:x86_64-linux => copy:duckdb_native:x86_64-linux:3.2.1
#0 2.026 (See full trace by running task with --trace)
-----
failed to solve: process "/bin/sh -c rake build" did not complete successfully: exit code: 1
```

Reply:
I updated `docker-compose v2.16.0` but got the same error... :cry:

Reply:
```
#0 2.025 /usr/bin/install -c -m 0755 duckdb_native.so /home/suke/mywork/ruby-duckdb/lib/duckdb
```
I'm looking into it and don't understand why the folder path here is `/home/suke/mywork/ruby-duckdb`. It looks like a path in your host machine instead of a path inside the image. 🤔 
The path in my environment is this:
```
#16 2.280 /usr/bin/install -c -m 0755 duckdb_native.so /root/ruby-duckdb/lib/duckdb
```

Reply:
I created my environment from scratch and works fine. Thank you.